test/tilequery_test.rb

```diff
@@ -0,0 +1,15 @@
+require 'mapbox'
+require 'test/unit'
+require 'mocha/setup'
+require 'shoulda'
+
+module Mapbox
+  class TilequeryTest < Test::Unit::TestCase
+    should "#tilequery" do
+      Mapbox.access_token = ENV["MapboxAccessToken"]
+      result = Mapbox::Tilequery.tilequery(
+        "mapbox.mapbox-streets-v7", {"longitude" => -100, "latitude" => 38})
+      assert result
```

Comment:
doesn't look like this particular test request returns any features and i think it might be because of radius being `required for queries against point and line data.` not totally sure here, but i would make the test one that actually returns some features.